### ISSUE TITLE
[REQUEST] Sort user repos by most stars/forks

### ISSUE THREAD

**Issue by kshvmdn** [slapperwan/gh4a#354]
It'd be cool if I could sort a user's repositories by most starred/forked (in addition to the current options). 

Not sure if the API allows for this, but if it does, I think it'd be a great feature.

**Comment by maniac103** [comment]
Unfortunately this isn't possible. Only creation date, update date, last push date or name are supported. See here: https://developer.github.com/v3/repos/#list-user-repositories

**Comment by evaryont** [comment]
Is this really dependent on Github implementing it server side? Octodroid does have the numbers, so it could sort the list itself. I don't know what kind of performance profile that would look like however.

**Comment by maniac103** [comment]
Yes, it is, because the repo list is paged and only loads 100 repos per page. Think of orgs like CyanogenMod with 1800 repos ... Octodroid doesn't have the numbers for the 1700 repos it didn't load yet ;)

**Comment by alorma** [comment]
@maniac103 that's not exactly true: https://developer.github.com/v3/search/#search-repositories

**Comment by maniac103** [comment]
@alorma The repo search doesn't support the other needed sort modes though. One of the many inconsistencies in Github's API...

**Comment by alorma** [comment]
I know... api is not so good...

Maybe you can use the search or repositories endpoint depending on which filter the user applies
